rules/falco_rules.yaml

```diff
@@ -475,7 +475,7 @@
     a shell configuration file was read by a non-shell program (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [file, mitre_discovery]
+  tags: [file, mitre_TA0007_discovery, mitre_TA0005_defense_evasion, mitre_T1562_impair_defenses, mitre_T1562.001_disable_or_modify_tools]
```

Comment:
same above

Reply:
Reading a shell config file could lead to defense tool discovery, hence `TA0007`, and from there the next (plausible) logical step to `T1562`. Maybe the connection between these two is weak?

Reply:
but read shell config file doesn't do anything to impair defense

Reply:
Solved